docs/admin/setup/kubernetes.rst

```diff
@@ -1,464 +0,0 @@
-Kubernetes Setup
-----------------
-
-This section describes how to set up an environment deployed in Kubernetes.
-
-.. note::
-
-   These steps were part of an experimental setup.
-   This does not result in a production-ready deployment.
-   There is ongoing work on a `Helm chart <https://github.com/ls1intum/artemis-helm>`__ to simplify the deployment of Artemis.
-
-
-**Prerequisites:**
-
```

Comment:
Why do we remove the complete Kubernetes section without adding it to the new documentation?

Reply:
As the documentation already states

> These steps were part of an experimental setup.

The setup steps were part of an experiment to split Artemis into multiple microservices rather than the current distributed approach. By now it is just outdated and no longer relevant.
The mentioned experimental Helm chart also has not been updated in over two years and is not described in this documentation at all with the exception of linking to it in the header.

-> Removed since no longer relevant. If the Helm chart will be maintained in the future, completely new documentation will be needed for that anyway.

---

todo for me: Remove the old (unchanged since 2021) Kubernetes files in src/main/kubernetes in a follow-up PR to remove the duplication with the Helm chart repo.

Reply:
I see, thank you for the explanation 👍